### PR TITLE
Create navigator

### PR DESCRIPTION
**Pull request by drewdrewthis**

### CODE REVIEW

File: src/templates/Navigator/index.js [hunk 1]
```diff
@@ -21,7 +50,11 @@ Navigator.propTypes = {
    * This prop will be add a new className to any inherent classNames
    * provided in the component's index.js file.
    */
-  className: React.PropTypes.string,
+  className: PropTypes.string,
+  /**
+   * Text to show up on left rail
+   */
+  txt: PropTypes.string
```
Comment (jmcolella) [inline src/templates/Navigator/index.js]: I think this prop name can be more descriptive

File: src/templates/Navigator/index.js [hunk 2]
```diff
@@ -1,16 +1,45 @@
...
+          >
+            { txt }
+          </TextComponent>
+        }
+      </div>
+      <div className={ styles.main }>
+        <StepProgress
+          className={ styles['step-progress'] }
+          data={[
+            { active: false, inactive: false, icon: 'icon-calculator', label: 'Aenean', link: '#' },
```
Comment (jmcolella) [inline src/templates/Navigator/index.js]: a bit unrelated, but I think `StepProgress` should have `onClick` handlers for each item. Would make this component more flexible for the end user

File: src/templates/Navigator/navigator.module.scss [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+.navigator {
+  // Place your component specific scss here.
+  background-color: $neutral-7;
+  height: 100%;
+  display: flex;
+}
+
+.left-rail {
+  align-self: flex-stretch;
+  width: ru(11);
```
Comment (drewdrewthis) [inline src/templates/Navigator/navigator.module.scss]: Not really.. if they don't give us a grid with the layout, I don't think we should bother applying it.

File: src/templates/Navigator/index.js (hunk 2 above)
Comment (drewdrewthis) [inline src/templates/Navigator/index.js]: Yeah, I agree UX wise, but that's something to discuss at the app level.

**Pull request merged**